tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/ITestCommand.cs

```diff
@@ -10,7 +10,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit;
 
 internal interface ITestCommand : IDuckType
 {
-    ITest Test { get; }
+    ITest? Test { get; }
 
-    ITestResult Execute(ITestExecutionContext context);
+    ITestResult Execute(ITestExecutionContext? context);
```

Comment:
This can't return `null`?

Does the destination method _allow_ passing `null`? If not, then we probably _shouldn't_ mark `ITestExecutionContext` as nullable? 🤔

Reply:
remove the nullable from the context, it requires it, also the method always returns an instance. Changed made in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)